ts/sagas/startup/watchLogoutSaga.ts

```diff
@@ -48,6 +49,8 @@ export function* logoutSaga(
     };
     yield* put(logoutFailure(logoutError));
   } finally {
+    // clean up crypto keys
+    yield* deleteTheCurrentKeyAndGenerateNewKeyTag();
```

Comment:
Since the new key is now re-generated at every login attempt, should we just remove the key (and rely on login generating the new key), just for speed reasons during logout?

Reply:
Here we recreate only a new `keyTag` not the key itself.